runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/widgets/ElementAwareAutocompleteWidget.tsx

```diff
@@ -286,8 +286,8 @@ const ElementAwareAutocompleteWidget: FC<ElementAwareAutocompleteProps> =
                     );
                 })
                 .concat(attendedProcessVariables)
-                .filter(variable => variable !== undefined)
-                .flatMap(variable => variable);
+                .flatMap(variable => variable)
+                .filter(variable => variable !== undefined);
```

Comment:
why? it increases operations

Reply:
changed order, of the filter and flatMap, because it didn't filter undefined (when I created an assign to global action, clicked on + to add a global variable, but went to other action I got error.

![image](https://user-images.githubusercontent.com/61866178/218696374-b8a10c34-9816-4e32-b343-67ad707f500a.png)

![image](https://user-images.githubusercontent.com/61866178/218696054-aea24fcf-986e-4b68-8faf-7ca635830207.png)